pkgs/by-name/by/bytecode-viewer/package.nix

```
@@ -0,0 +1,75 @@
+{ lib
+, fetchFromGitHub
+, jre
+, makeWrapper
+, maven
+, icoutils
+, copyDesktopItems
+, makeDesktopItem
+}:
+
+maven.buildMavenPackage rec {
+  pname = "bytecode-viewer";
+  version = "2.12";
+
+  src = fetchFromGitHub {
+    owner = "Konloch";
+    repo = pname;
```

https://github.com/NixOS/nixpkgs/issues/277994
```suggestion
    repo = "bytecode-viewer";
```